mono/utils/mono-logger-internals.h

```diff
@@ -17,6 +17,7 @@ typedef enum {
 	MONO_TRACE_THREADPOOL		= (1<<7),
 	MONO_TRACE_IO_THREADPOOL	= (1<<8),
 	MONO_TRACE_IO_LAYER		= (1<<9),
+	MONO_TRACE_W32HANDLE	= (1<<10),
```

Comment:
Why do we want to split io-layer from w32handle?

This is a minor breaking change and requires 2 things:

- updating the docs (man pages, --help, website)
- adding a section on the release notes on this
- review mentions of io-layer on the website for possible further edits